src/spotify.cpp

```diff
@@ -684,6 +684,23 @@ void Spotify::sendCommand(const QString& type, const QString& entityId, int comm
                 }
             }
         }
+    } else if (command == MediaPlayerDef::C_QUEUE) {
```

Comment:
This gives a compile error:
```
../src/spotify.cpp:687:43: error: 'C_QUEUE' is not a member of 'MediaPlayerDef'
     } else if (command == MediaPlayerDef::C_QUEUE) {
                                           ^~~~~~~
```
MediaPlayerDef is part of the shared integrations.library project, which in this case also requires a PR